lib/model/QubitMenu.php

```diff
@@ -103,6 +104,11 @@ public function getPath($options = [])
             $path = $url;
         }
 
+        if (isset($options['removeIndex']) && true === $options['removeIndex']) {
+            $regex_for_index = '/\A(\/index.php)/';
+            $path = preg_replace($regex_for_index, '', $path);
+        }
+
```

Comment:
I wonder if it would be better to do this in https://github.com/artefactual/atom/blob/qa/2.x/lib/routing/QubitMetadataRoute.class.php#L42-L213 or if that would have other implications. I wouldn't know why, but maybe sfRoute is not considering the script name in `matchesUrl`.

Reply:
Yup, I can confirm that sfRoute isn't considering the script name in it's implementation of `matchesUrl`. I wasn't sure if that would have impact elsewhere so I ended up making the change in the menu to limit which places would trim `index.php` from the URL.